README.md

```diff
@@ -14,21 +14,21 @@ Then follow the instructions below to set up `calyx` and `odgi`.
 ### Installing Dependencies
 
 #### Calyx
-Follow these [instructions](https://docs.calyxir.org/) to install calyx. You will need to configure the calyx driver, `fud`. We recommend using the native calyx interpreter, so once `fud` is set up, run
+Follow these [instructions](https://docs.calyxir.org/) to install calyx. You will need to complete the Installing the Commandline Driver section, but can skip Running the Core Tests. We recommend using the native calyx interpreter, so once `fud` is set up, run
 ```
-fud config stages.interpreter $ABSOLUTE_PATH_TO_CALYX_REPO/target/debug/interp
+fud config stages.interpreter <full path to Calyx repository>/target/debug/interp
 ```
-where `$ABSOLUTE_PATH_TO_CALYX_REPO` is the absolute path to the root directory. For example, if you downloaded calyx in `/Users/username/project`, you would run `fud config stages.interpreter /Users/username/project/target/debug/interp`.
+where `<full path to Calyx repo` is the absolute path to the root directory. For example, if you downloaded calyx in `/Users/username/project`, you would run `fud config stages.interpreter /Users/username/project/calyx/target/debug/interp`.
```

Comment:
where `<full path to Calyx repository>` is the absolute path to the root directory of the calyx project.

Reply:
Yep. Good catch!